CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#4710](https://github.com/thanos-io/thanos/pull/4710) Store: add metric to capture timestamp of the last loaded block.
 - [#4736](https://github.com/thanos-io/thanos/pull/4736) S3: Add capability to use custom AWS STS Endpoint.
 - [#4764](https://github.com/thanos-io/thanos/pull/4764) Compactor: add `block-viewer.global.sync-block-timeout` flag to set the timeout of synchronization block metas.
+- [#4827](https://github.com/thanos-io/thanos/pull/4827) Compactor: add a new alert `ThanosStoreObjstoreOperationBlocksMissing` to flag not even a single block has been uploaded during last 6 hours.
```

Comment:
Is this alert related to the compactor?

Reply:
It is not to Compactor.. I have fixed this.